test/functional/mempool_package_rbf.py

```diff
@@ -169,22 +169,34 @@ def test_package_rbf_additional_fees(self):
 
         self.log.info("Check replacement pays for incremental bandwidth")
         package_hex3, package_txns3 = self.create_simple_package(coin, parent_fee=DEFAULT_FEE, child_fee=DEFAULT_CHILD_FEE)
+        package_3_size = sum([tx.get_vsize() for tx in package_txns3])
+        incremental_sats_required = Decimal(package_3_size) / COIN
+        incremental_sats_short = incremental_sats_required - Decimal("0.00000001")
+        # Recreate the package with slightly higher fee once we know the size of the new package, but still short of required fee
+        package_hex3, package_txns3 = self.create_simple_package(coin, parent_fee=DEFAULT_FEE, child_fee=DEFAULT_CHILD_FEE + incremental_sats_short)
+        assert_equal(package_3_size, sum([tx.get_vsize() for tx in package_txns3]))
         pkg_results3 = node.submitpackage(package_hex3)
-        assert_equal(f"package RBF failed: insufficient anti-DoS fees, rejecting replacement {package_txns3[1].rehash()}, not enough additional fees to relay; 0.00 < 0.00000{sum([tx.get_vsize() for tx in package_txns3])}", pkg_results3["package_msg"])
-
+        assert_equal(f"package RBF failed: insufficient anti-DoS fees, rejecting replacement {package_txns3[1].rehash()}, not enough additional fees to relay; {incremental_sats_short} < {incremental_sats_required}", pkg_results3["package_msg"])
         self.assert_mempool_contents(expected=package_txns1)
+
+        package_hex3_1, package_txns3_1 = self.create_simple_package(coin, parent_fee=DEFAULT_FEE, child_fee=DEFAULT_CHILD_FEE + incremental_sats_required)
```

Comment:
nit: variable naming could be more clear, e.g. `fail_package_hex3` and `success_package_hex3` instead of just numbers

Reply:
done